File: C++/8.cpp
/*
Faça um programa que solicite um número ao usuário, seu programa deve garantir que o número 
digitado é maior que 1. Por fim, o programa deve exibir dois resultados:
a.	A soma de todos os números entre 1 e o número digitado (incluindo 1 e o número digitado 
na soma);
b.	A média de todos os números entre 1 e o número digitado (incluindo 1 e o número digitado 
na soma).
*/

#include <iostream>

using namespace std;

int main(){
    int numeroDigitado, i, qtdNumeros=0;
    int soma = 0, media;
    bool trueorfalse = false;

    while(trueorfalse == false){
        cout << "Digite um numero maior que 1: ";
        cin >> numeroDigitado;
        if(numeroDigitado <= 1){
            cout << "Voce digitou um numero menor/igual a 1.Tente novamente.\n";
        }else{
            trueorfalse = true;
        }   
    }
    for (i = 1; i <=numeroDigitado; i++){
        soma += i;    
        qtdNumeros += 1;
    }
    media = soma / qtdNumeros;

    cout << "A soma de todos os numeros entre 1 e " << numeroDigitado << " eh: " << soma;
    cout << "\nA media de todos os numeros entre 1 e " << numeroDigitado << " eh: " << media;

    return 0;
}

File: Python/8.py
"""
Faça um programa que solicite um número ao usuário, seu programa deve garantir que o número
digitado é maior que 1. Por fim, o programa deve exibir dois resultados:
a.	A soma de todos os números entre 1 e o número digitado (incluindo 1 e o número digitado
na soma);
b.	A média de todos os números entre 1 e o número digitado (incluindo 1 e o número digitado
na soma).
"""

while True:
    numeroSolicitado = int(input("Digite um número maior que 1: "))
    if numeroSolicitado > 1:
        break

soma = 0
qntdNum = 0
for contador in range(1, numeroSolicitado+1):
    soma += contador
    qntdNum += 1

media = soma/qntdNum

print(f"A soma de todos os números entre 1 e {numeroSolicitado} é {soma}")
print(f"A média de todos os números entre 1 e {numeroSolicitado} é {media}")
